src/ria/main.c

```diff
@@ -58,6 +59,7 @@ static void init()
     mou_init();
     rom_init();
     led_init();
+    rt_clock_init();
```

Comment:
```suggestion
    rtc_init();
```

Reply:
The `#include "hardware/rtc.h"` contains `void rtc_init(void);` so I don't think we can use that name. Is there a way to do it or a different name we can use?

Reply:
This and the ntp code in one driver wouldn't be too much. So ntp.c? Unless you got a better TLA?